src/app/components/NavBar.tsx

```diff
@@ -33,7 +33,14 @@ const NavItems: { path: string; title: string }[] = [
 
 export function Navbar() {
   return (
-    <nav className="flex items-center justify-between w-full !h-[74px] !min-h-[74px] z-[100]">
+    <nav
+      className={
+        "sticky top-0 sm:border-b-0 border-b-2 sm:border-none border-gray-800 sm:static " +
+        "bg-base-200 flex items-center justify-between w-full !h-[74px] !min-h-[74px] " +
```

Comment:
I just realized that the navbar should be transparent when on desktop and on the landing page, this looks a bit weird:

<img width="1334" alt="Bildschirmfoto 2024-05-23 um 22 40 38" src="https://github.com/DeXter-on-Radix/website/assets/44790691/2f9709ca-b395-4a9f-a326-19d69cd23d78">

I think to solve it it is sufficient to only add the background color for mobile screens (<640px). I think this should fix the issue:

```suggestion
        "bg-base-200 sm:bg-transparent flex items-center justify-between w-full !h-[74px] !min-h-[74px] " +
```